src/core.js

```diff
@@ -748,7 +748,7 @@ define([
         if (target) {
             // the .change fires the validation controls
             if (!widgets.util.getWidget(target, this).options.noRichText &&
-                ((!mug && _this.data.core.form.useRichText !== false &&
+                ((!mug && _this.data.core.form.noRichText !== false &&
```

Comment:
https://github.com/dimagi/Vellum/pull/667/commits/ef2d597eda1c4a615e2d86111e6535e67b0b215c - got rid of the mug checks, but still need to check the form property